assignments/function-conversion.js

```diff
@@ -4,12 +4,18 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+let myFunction = () => {console.log('Function was invoked!')};
+myFunction();
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {return param;};
+anotherFunction(2);
+
```

Comment:
could do an implicit return here that looks like this:
```
let anotherFunction = param => param;
```
when an arrow function only has one parameter being passed to it and the return statement is just one line, you don't have to use the curley braces or the return statement.